test/problems/interactivemultipass/problem.yaml

```diff
@@ -1,8 +1,12 @@
 name: interactive multi-pass
-author: Michael Zündorf
-source:
-source_url:
 uuid: 42c9ed2e-f579-46ac-ae2c-191069f3df70
 license: unknown
 rights_owner:
-validation: custom interactive multi-pass
+problem_format_version: 2023-07-draft
+type:
+  - pass-fail
```

Comment:
Isn't `pass-fail` implied when not given?

Reply:
yes, but i prefer `bt upgrade` to be explicit about this.